app/src/androidTest/java/com/sdp/movemeet/Backend/BackendActivityManagerDebugTest.java

```diff
@@ -34,6 +34,9 @@
     @Rule
     public ActivityScenarioRule<BackendActivityManagerDebug> testRule = new ActivityScenarioRule<>(BackendActivityManagerDebug.class);
 
+    @Test
```

Comment:
What is this test for ?

Reply:
Oops, I forgot again to delete it, I will do it now 0:-)